docker/release.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
 
-vt_base_version='v13.0.0'
+vt_base_version='14.0.5-SNAPSHOT'
```

Comment:
This doesn't seem right, it should be `v14.0.5`.

Reply:
Like the other files that have the version it is kept at `-SNAPSHOT`. The release script changes it to the release version and then back to a snapshot version